modules/openapi-generator/src/main/resources/rust/Cargo.mustache

```diff
@@ -32,14 +32,13 @@ homepage = "{{.}}
 {{/homePageUrl}}
 
 [dependencies]
-serde = "^1.0"
-serde_derive = "^1.0"
+serde = { version = "^1.0", features = ["derive"] }
```

Comment:
Using `serde` with the `derive` feature instead of `serde_derive`

Reply:
Note: `serde_derive` hasn't been required in years at this point :) Even the `serde` docs show using the `derive` feature: https://serde.rs/derive.html#using-derive